app/forms/add_to_order_form.rb

```diff
@@ -25,7 +25,28 @@ def initialize(original_order)
   def save
     raise(ActiveRecord::RecordInvalid, self) unless valid?
 
-    add_to_order!
+    @facility_id = @facility_id&.to_i
+
+    order_project = Projects::Project.find_by(
+      name: "#{current_facility.abbreviation}-#{original_order.id}",
+      facility_id: current_facility.id
+    )
```

Comment:
I think we will need an association between the original_order and the cross-core project:
`belongs_to :cross_core_project`